src/components/ui/Modal/Modal.stories.tsx

```diff
@@ -77,7 +77,7 @@ export const BasicExample = () => {
               <ModalHeader title="Start an instant meeting"/>
 
               <ModalBody>
-                <p style={{ 'color': '#616672', 'fontSize': '14px','margin': '0 0 0.5rem' }}>
+                <p style={{'fontSize': '14px','margin': '0 0 0.5rem',  'lineHeight': '1.43'}}>
```

Comment:
Are we only trying to update the story? Should we change the styling of the component itself?

Reply:
Yah good point! Updated!